web/src/main/webapp/xslt/base-layout.xsl

```diff
@@ -80,6 +80,12 @@
             </div>
           </xsl:when>
           <xsl:otherwise>
+            <script type="text/javascript">
+              //detect ie11 browser; display warning
+              if (/*@cc_on!@*/false || !!document.documentMode) {
+                document.body.innerHTML += '<div class="alert alert-warning">This application is not available in this browser. Please use a recent version of Edge, Firefox, Chrome, Safari or Opera in stead. Or alternatively use the limited <a href="../search">no-javascript interface</a></div>';
```

Comment:
"instead", not "in stead"

Reply:
thanx @archaeogeek 
the approach of @zhuoyuezhou made me think, why do we have this lib in the production system?

@juanluisrp suggested to remove line https://github.com/geonetwork/core-geonetwork/blob/master/web-ui/src/main/resources/WEB-INF/classes/web-ui-wro-sources.xml#L36 

And indeed this also fixes the problem, i'm going to change the PR accordingly

Reply:
@pvgenuchten 
Look like you are reverting the commit 
https://github.com/geonetwork/core-geonetwork/commit/3b9fda9923b29c8bd18158aa6ce35d393e9be917#diff-b63d73df392776a0eb4097d238004302 from https://github.com/geonetwork/core-geonetwork/pull/4431 from @julsbreakdown 

Should you be removing 2 lines? Or will the openlayers be broken in ie11?  And if fixing ie11 will you be breaking openlayers?